cumulusci/cli/cci.py

```diff
@@ -1004,6 +1004,8 @@ def task_run(config, task_name, org, o, debug, debug_before, debug_after, no_pro
         # Unexpected exception; log to sentry and raise
         handle_exception_debug(config, debug, no_prompt=no_prompt)
 
+    config.alert("Task complete")
```

Comment:
Is this going to start beeping at me when I run the tests for `task_run` and `flow_run`?

Reply:
ran the full suite with both py.test and nose, and both suppressed the bell, presumably because they capture stdout.

Reply:
I often run nose with the -s flag to un-suppress stdout while working on tests...

Reply:
resolution: after some testing, though we're not sure *why*, even with -s, the task_run and flow_run tests don't cause the bell. so not super alarmed about this. but if we ever need to, let's just wrap the echo in a partial so its mockable.